### ISSUE TITLE
Keyboard problem (key "4")

### ISSUE THREAD

**Issue by bepolymathe** [Guake/guake#1033]
I have a very strange problem with Guake. I'm under fedora 27 (wayland/gnome 3.26) with a bepo keyboard layout (but it doesn't change if I change the layout).

When I start Guake, about half the time, the number "4" is not recognized. I can't even type it. The cursor blinks but does not display any characters. If I leave Guake and start over, it works normally.

It's very painful because it's random.
Do you have any idea where the problem came from?
Thank you in advance.

**Comment by bepolymathe** [comment]
And this the behaviour after sleep mode...

![peek 2017-11-27 14-50](https://user-images.githubusercontent.com/29939805/33269864-6e6de27a-d382-11e7-9e81-22cd0278e4f2.gif)

**Comment by egmontkob** [comment]
The cursor becomes outlined rectangle for a short time when you press 4. This means that the Guake window loses focus. Looks like a global window manager shortcut or something similar outside of Guake, the keypress 4 probably doesn't even reach Guake.

**Comment by bepolymathe** [comment]
Yes, I thought of that but I don't have any other keyboard shortcuts corresponding to Shift+4. 
Besides, if I block capital letters, it works again...

**Comment by bepolymathe** [comment]
This problem persists. Do you have any idea ? It's very painful.

**Comment by Stibbons** [comment]
yes, look like a shortcut on your system. i cannot reproduce here
